src/tic_tac_toe/console.clj

```diff
@@ -1,10 +1,34 @@
 (ns tic-tac-toe.console)
 
+(defn- display-to-console [message]
+  (print message))
+
+(defn- print-new-line []
+  (print "\n"))
+
 (defn print-empty-board []
-  (print "---\n---\n---\n"))
+  (display-to-console "---\n---\n---\n"))
+
+(defn print-board [board]
+  (display-to-console (board 0))
```

Comment:
loads of duplication here, can you not just print each row rather than each cell?